AirCasting/Settings/BackendSettingsView.swift

```diff
@@ -58,7 +59,8 @@ struct BackendSettingsView: View {
             urlProvider.baseAppURL = url ?? URL(string: "http://aircasting.org/api")!
             presentationMode.wrappedValue.dismiss()
             do {
-                try logoutController.logout()
+                isShowingLoadingScreen = true
+                try logoutController.logout(onEnd: { isShowingLoadingScreen = false })
```

Comment:
onEnd is trailing closure, can be done without specifying parameter: `logoutController.logout() { isShowingLoadingScreen = false }`

Reply:
I thought it is nice to read onEnd but I will change it

Reply:
> I thought it is nice to read onEnd but I will change it

Of course it is a matter of taste, but generally iOS folks do use the trailing syntax whenever possible. However if you feel this is more readable I am just fine with leaving it as is :)